### PR TITLE
Add Decision maker field

### PR DESCRIPTION
**Pull request by dhaselhan**

### CODE REVIEW

File: alcs-api/config/default.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "PASSWORD": ""
   },
   "REDIS": {
-    "URL": "redis://redis:@localhost:6379"
+    "URL": "redis://:redis@localhost:6379"
```
Comment (mhuseinov) [inline alcs-api/config/default.json:31]: thanks!

File: alcs-api/src/application/application-decision-maker/application-decision-maker.service.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { ApplicationDecisionMaker } from './application-decision-maker.entity';
+
+@Injectable()
+export class ApplicationDecisionMakerService {
+  constructor(
+    @InjectRepository(ApplicationDecisionMaker)
+    private readonly applicationTypeRepository: Repository<ApplicationDecisionMaker>,
```
Comment (mhuseinov) [inline alcs-api/src/application/application-decision-maker/application-decision-maker.service.ts:10]: I thought we getting read of this 'readonly'

File: alcs-api/src/application/application.controller.ts [hunk 3]
```diff
@@ -67,10 +71,15 @@ export class ApplicationController {
...
+      ? await this.applicationDecisionMakerService.get(
+          application.decisionMaker,
+        )
+      : undefined;
     const app = await this.applicationService.createOrUpdate({
       ...application,
       type,
-      typeUuid: type.uuid,
+      decisionMaker,
       title: 'Do we still need title?',
```
Comment (mhuseinov) [inline alcs-api/src/application/application.controller.ts:83]: Did we ever get response on this one or we waiting for wireframes?

File: alcs-api/src/providers/typeorm/migrations/1659644464368-seed_application_decision_maker.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    ('76c1d7d2-0a3a-4649-98bd-6b66305eecb1', NULL, '2022-08-04 20:28:41.109707+00', '2022-08-04 20:28:41.109707+00', 'alcs-api', NULL, 'OKAN', 'Okanagan Panel', 'Okanagan Panel'),
+    ('7713b381-af54-4e1d-bffb-515dec2210cc', NULL, '2022-08-04 20:27:32.698091+00', '2022-08-04 20:27:32.698091+00', 'alcs-api', NULL, 'INTE', 'Interior Panel', 'Interior Panel'),
+    ('acfb734d-5194-4c05-8e9f-ff87a99c46e9', NULL, '2022-08-04 20:30:05.420682+00', '2022-08-04 20:30:05.420682+00', 'alcs-api', NULL, 'SOUT', 'South Coast Panel', 'South Coast Panel'),
+    ('ae8cc147-ca3d-4bf4-8856-93097ed4acbb', NULL, '2022-08-04 20:28:41.109707+00', '2022-08-04 20:28:41.109707+00', 'alcs-api', NULL, 'ISLE', 'Island Panel', 'Island Panel'),
+    ('fd6dce41-d1a7-4776-87d9-ddba75e33676', NULL, '2022-08-04 20:28:41.109707+00', '2022-08-04 20:28:41.109707+00', 'alcs-api', NULL, 'NORT', 'North Panel', 'North Panel');
+    `);
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`delete from public.holiday_entity`);
```
Comment (mhuseinov) [inline alcs-api/src/providers/typeorm/migrations/1659644464368-seed_application_decision_maker.ts:23]: application_decision_maker

**Review by mhuseinov — APPROVED** [review]

File: alcs-api/src/application/application-decision-maker/application-decision-maker.service.ts (hunk 2 above)
Comment (dhaselhan) [reply alcs-api/src/application/application-decision-maker/application-decision-maker.service.ts]: Woops, removed.

File: alcs-api/src/application/application.controller.ts (hunk 3 above)
Comment (dhaselhan) [reply alcs-api/src/application/application.controller.ts]: None yet, I've gone ahead and deleted it and replace the title with the same as the card fileNumber (applicant)

File: alcs-api/src/providers/typeorm/migrations/1659644464368-seed_application_decision_maker.ts (hunk 4 above)
Comment (dhaselhan) [reply alcs-api/src/providers/typeorm/migrations/1659644464368-seed_application_decision_maker.ts]: Fixed!

**Pull request merged**